### ISSUE TITLE
Evolving Losses for Unsupervised Video Representation Learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#59]
# TL;DR
They proposed a mechanism to aggregate information into a network of RBG video inputs using task balancing and distillation with evolutionary algorithms while allowing multi-task and multimodal unsupervised learning, and achieved very high accuracy in an action detection task using 2 million videos.
![image](https://user-images.githubusercontent.com/24524018/91716239-34348600-ebca-11ea-8fd0-35236225d6cc.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2002.12177

# Submission Dates(yyyy/mm/dd)
2020/02/26

# Authors and institutions

AJ Piergiovanni, Anelia Angelova, Michael S. Ryoo
- google

# Methods

# Results

# Comments
